src/test/java/academy/softserve/edu/asserts/AbstractElementAssert.java

```diff
@@ -309,6 +321,90 @@ public final AbstractElementAssert selectedDropdownEqualsIgnoreCase(final String
         return this;
     }
 
+
+    public final AbstractElementAssert isMoreThan(int quantity) {
```

Comment:
rename quantityIsMoreThan()
re-do log messages: Quantity of elements should be more than ...